docs/en/observability/logs-stream.asciidoc

```diff
@@ -143,10 +143,104 @@ If you're not seeing your log files in {kib}, check the following in the `elasti
 If you're still running into issues, see {fleet-guide}/fleet-troubleshooting.html[{agent} troubleshooting] and {fleet-guide}/elastic-agent-configuration.html[Configure standalone Elastic Agents].
 
 [discrete]
-[[logs-stream-whats-next]]
-== What's next?
+[[logs-stream-enhance-logs]]
+== Get the most out of your log data
 
-For more information on deploying and managing logs in Elastic Observability, see the following links:
+Now that you've sent your log data to {es}, this section helps you make sure you're getting the most out of it. You can do this by taking your unstructured log data and using ingest pipelines to extract structured fields. Extracting structured fields from your log data makes it easier to search, analyze, and filter your log data. 
 
-- The <<logs-checklist>> consolidates links to documentation on sending log data, configuring logs, and analyzing logs.
-- <<monitor-logs>> has information on visualizing and analyzing logs.
\ No newline at end of file
+Let's look at the following log data example:
+
+[source,log]
+----
+2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.
+----
+
+This log is made up of the following potential fields:
+
+- *timestamp*: "2023-08-08T13:45:12.123Z"
+- *log.level*: "WARN"
+- *host.ip*: "192.168.1.101"
+- *message*: "Disk usage exceeds 90%."
+
+Extracting these structured fields from your log data lets you search or filter your data in the following ways:
+
+- *timestamp* – Extracting this field lets you sort logs by date and time. This is helpful when you want to view your logs in the order that they occurred or identify when issues happened.
```

Comment:
lets mention that these are also ECS fields which should be the same across all documents which allows correlation.